setup.py

```diff
@@ -71,6 +71,7 @@ def finalize_options(self):
     },
     extras_require={
         'backend': ['onnx>=1.2.3'],
+        'numpy': ['numpy>=1.15.0']
```

Comment:
Based on the threads in Issues, I though 1.16 was required and not just 1.15

e.g. https://github.com/Microsoft/onnxruntime/issues/510

Reply:
Actually numpy has good backward compatibility. I tested the one built with numpy1.15.0 locally with 1.15.1 and 1.16.1 on my boxes (linux/windows) it works for both. 

And from feedback to converter team, 1.15.x seems to be the most popular one in the issues filed. So 1.15.0 seems to be a reasonable lowest version we can build with and specify as extra package requirement